downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -84,6 +84,14 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 - Introduced GraphQL server-side caching support
 
+#### `crypto` package
+
+- Introduced new APIs for ML-KEM-768(Kyber768) key encapsulation mechanism.
```

Comment:
Unless there's a specific reason not to, shall we add space before (?
```suggestion
- Introduced new APIs for ML-KEM-768 (Kyber768) key encapsulation mechanism.
```

Reply:
Fixed in [78b1103](https://github.com/ballerina-platform/ballerina-dev-website/pull/8892/commits/78b110378dbf2ec76f4702e6ccfd5df857239bc8)